src/containerapp/azext_containerapp/containerapp_env_decorator.py

```diff
@@ -167,8 +164,7 @@ def construct_payload(self):
         self.managed_env_def["tags"] = self.get_argument_tags()
         self.managed_env_def["properties"]["zoneRedundant"] = self.get_argument_zone_redundant()
 
-        if self.get_argument_enable_workload_profiles() is True:
-            self.managed_env_def["properties"]["workloadProfiles"] = get_default_workload_profiles(self.cmd, self.get_argument_location())
+        self.managed_env_def["properties"]["workloadProfiles"] = get_default_workload_profiles(self.cmd, self.get_argument_location())
```

Comment:
why do you remove the argument check? If it is removed, the --enable-wp flag will not work, right?